### PR TITLE
Gcrone/connection service

### PR DESCRIPTION
**Pull request by gcrone**
This update removes the actual connection server from the microservices package and loads it from its own package (see Issue #32).

- The `connection-service/Dockerfile` does a `git clone` from the `connectionserver` repo and installs it as a python package.
- The `connection-service/Dockerfile` uses `almalinux:9` since the python version under CentOS8 is too old to run the current `flask.py`.

### CODE REVIEW

**Review by bonnieking — APPROVED** [review]
looks good to me

File: connection-service/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
- && yum -y install python3-devel \
+ && yum -y install python3-pip python3-devel git \
  && yum clean all
 
-COPY connection-flask.py /
-COPY requirements.txt /
 
-RUN pip3 install --upgrade pip
-RUN pip3 install -r /requirements.txt
+RUN git clone --depth 1 https://github.com/DUNE-DAQ/connectivityserver.git
```
Comment (plasorak) [inline connection-service/Dockerfile]: I think this is fine, however for the future, could this be made such that it uses the correct version of the connectivity service GitHub? We probably don't want to clone `develop`'s head every time we rebuild the image. So for good measure, I'd suggest tagging the connectivity service repo (format is `vX.X.X`) and adding it here with a: `-b the-version`.

**Review by plasorak — CHANGES_REQUESTED** [review]
Looks good, I just suggest it be a bit more strict with versioning.

File: connection-service/Dockerfile (hunk 1 above)
Comment (gcrone) [reply connection-service/Dockerfile]: Good point, I've updated it to specify v1.0.0

**Review by plasorak — APPROVED** [review]
Yep! all good now thanks!

**Pull request merged**